third-culture-software/bhima
# Update the inventory and stock imports using CSV files

Updated both importers for inventories and for stock.  Improved documentation.
Fixed a bug in the Inventory Registry search modal.

**TESTING**
- Use bhima_test
- Go to:  Inventory > Inventory Registry > [Menu] > Import Inventories
  - Select this file:   <bhima>/test/fixtures/inventory-to-import.csv  (review it first)
  - Click on the [Submit] button at the bottom of the form to import the inventories
- Go to: Inventories > Inventories List
   - Updated the default search param 'limit' to a large number (eg, 10000)
   - Enable the filter
   - In the 'Code' column filter enter:  Z1
   - Notice the two new inventory items that were just imported
- Go to: Stock > Stock Import
   - Select this file:  <bhima>/server/resources/templates/import-stock-template.csv (review it first)
   - Click [Submit] to import the 2 stock items
- Go to Stock > Lots (this should happen automatically)
   - Notice the 2 new lots (QNN2020, YAMOTO1)
- Visit the Assets Management > Assets Registry
  - Notice that there three assets (MOT1, MOT2, and YAMOTO1)
- Feel free to use the guidance to create new import files.
